src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables) {
+        String concat = "";
+        for (Printable p : printables) {
+            concat += p.getPrintableString() + " ";
```

Comment:
:bulb: *Java tip*: Java 8 has a `StringJoiner` class that can join multiple strings using a delimiter e.g. `aaa` `bbb` `ccc` --> `aaa, bbb, ccc` https://www.mkyong.com/java8/java-8-stringjoiner-example/